### PR TITLE
[3.0] Remove mentions of Symfony1

### PR DESCRIPTION
**Pull request by wouterj**
I think with Symfony 3, we should no longer document how Symfony >1 differs from Symfony 1.

| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | 3.0+ |
| Fixed tickets | - |

### CODE REVIEW

File: cookbook/event_dispatcher/before_after_filters.rst [hunk 1]
```diff
@@ -8,10 +8,10 @@ It is quite common in web application development to need some logic to be
 executed just before or just after your controller actions acting as filters
 or hooks.
 
-In symfony1, this was achieved with the preExecute and postExecute methods.
-Most major frameworks have similar methods but there is no such thing in Symfony.
-The good news is that there is a much better way to interfere with the
-Request -> Response process using the :doc:`EventDispatcher component </components/event_dispatcher/introduction>`.
+Most major frameworks have methods like ``preExecute()`` and ``postExecute()``,
```
Comment (javiereguiluz) [inline cookbook/event_dispatcher/before_after_filters.rst]: `Most major frameworks have [...]` -> `Some web frameworks define [...]` ?

**Comment by javiereguiluz** [comment]
:+1:

File: cookbook/event_dispatcher/before_after_filters.rst (hunk 1 above)
Comment (OskarStark) [inline cookbook/event_dispatcher/before_after_filters.rst]: :+1:

**Comment by WouterJ** [comment]
Thanks @javiereguiluz. I've applied your proposal.

**Comment by xabbuh** [comment]
looks good :+1:

**Pull request merged**